Readme.md

```diff
@@ -30,7 +30,7 @@ You can see a list of contributors [here](https://github.com/JohnCoates/Aerial/g
 1. Quit **System Preferences**.
 2. [Download the latest release of Aerial.saver.zip](https://github.com/JohnCoates/Aerial/releases/latest).
 
-    **If you are a new user**, Apple has just added some new videos and made some changes that can confuse the current build of Aerial. Because of this, you may see duplicates in the user interface. We suggest you [give the current beta (1.4.6beta5+) a try instead](https://github.com/JohnCoates/Aerial/releases), as this version will properly handle the duplicate issue (you can find more information on changes on [this issue](https://github.com/JohnCoates/Aerial/issues/630)).
+    **If you are a new user**, Apple has just added some new videos and made some changes that can confuse the current build of Aerial. Because of this, you may see duplicates in the user interface. We suggest you [give the current beta (1.4.6beta6) a try instead](https://github.com/JohnCoates/Aerial/releases), as this version will properly handle the duplicate issue (You can find more information on changes on [this issue](https://github.com/JohnCoates/Aerial/issues/630)).
```

Comment:
Technically, you shouldn't capitalize the first word in a parenthetical. Removing the parentheses and making it a separate sentence would warrant a capitalized "You", if desired.